kubeflow-katib.yaml

```diff
@@ -0,0 +1,80 @@
+package:
+  epoch: 0
+  name: kubeflow-katib
+  version: 0.15.0
+  description: Kubeflow Katib services
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - wget
+      - ca-certificates
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
+      - python-3.10-dev
+      - py3.10-pip
+      - gfortran
+      - pcre-dev
+      # libopenblas-dev liblapack-dev not sure if these are needed, builds fine without.  From https://github.com/kubeflow/katib/blob/b9dc63efb55bec19bdd80ca6f2930a65795f7c41/cmd/earlystopping/medianstop/v1beta1/Dockerfile#L10
```

Comment:
Upstream has libopenblas-dev and liblapack-dev during the build but not sure if we need them in wolfi as the builds are all working?